specification/keyvault/data-plane/TypeSpec/models.tsp

```diff
@@ -0,0 +1,962 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Foundations;
+
+namespace KeyVaultClient;
+
+interface Operations extends Azure.ResourceManager.Operations {}
```

Comment:
Switch to: https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/interfaces#Azure.Core.ResourceOperations